Scripts/script-BuildEWSQuery.yml

```diff
@@ -2,7 +2,13 @@ commonfields:
   id: BuildEWSQuery
   version: -1
 name: BuildEWSQuery
+releaseNotes: "Add paramter for stripping the subject from prefixes"
 script: |-
+  import re
```

Comment:
Since you have a script for this
You can easily right a test playbook for this specific script